packages/experimental/Shadow/src/__tests__/Shadow.test.tsx

```diff
@@ -14,7 +14,9 @@ const TestShadow: React.FunctionComponent<ShadowTestProps> = (props: ShadowTestP
   const theme = useFluentTheme();
   return (
     <Shadow shadowToken={theme.shadows[props.depth]}>
-      <Text>{props.displayText}</Text>
+      <View>
+        <Text>{props.displayText}</Text>
+      </View>
```

Comment:
For my own sake, could you add a test case where a Shadow wraps a `Pressable` from React Native? That _should_ be a View but I wanna make sure